### ISSUE TITLE
Installation fails on python3.13

### ISSUE THREAD

**Issue by matan1008** [tsukumijima/netifaces-plus#6]
Hey there!
The installation fails on python3.13 with ubuntu 24.04.

```
> py -m pip install --user netifaces-plus
ERROR: Could not find a version that satisfies the requirement netifaces-plus (from versions: none)
ERROR: No matching distribution found for netifaces-plus
```

**Comment by ntamas** [comment]
`netifaces-plus` is not yet compiled and published for Python 3.13, but the project itself builds and works just fine for me with that Python version. We need to wait for @tsukumijima to make a new release on PyPI with pre-built wheels for Python 3.13.

@tsukumijima Do you have time to make a new release in the near future?

**Comment by tsukumijima** [comment]
@ntamas  @matan1008 
I no longer use this library, moving to ifaddr, which is written entirely in Python.
So I don't recommend it, but it seems to still be used somewhere, so I've released a pre-built wheel for Python 3.13.
I haven't tested it, but it should work.
https://pypi.org/project/netifaces-plus/0.12.3/#files
